suse_migration_services/migration_config.py

```diff
@@ -43,8 +46,32 @@ def __init__(self):
             Defaults.get_migration_config_file()
         self.migration_custom_file = \
             Defaults.get_system_migration_custom_config_file()
-        with open(self.migration_config_file, 'r') as config:
-            self.config_data = yaml.safe_load(config)
+        self.config_data = self._parse_config_file(self.migration_config_file)
+
+    def _parse_config_file(self, config_file):
+        config_data = {}
+        with open(config_file, 'r') as config:
+            try:
+                validator = Validator(schema)
+                config_data = yaml.safe_load(config)
+                if config_data is None:
+                    # Config file is empty (or all comments), reset to empty dict
+                    config_data = {}
```

Comment:
Can return here, an empty config is accepted as valid.

Reply:
You're right, but that'd make it `return {}` which IMO looks weird :-) so I've left this as is